subcommand/lifecycle-sidecar/command_test.go

```diff
@@ -1,11 +1,16 @@
 package subcommand
 
 import (
+	"fmt"
+	"github.com/hashicorp/consul/agent"
```

Comment:
Imports are usually grouped as standard library imports, line break, other imports. It would be good to do the same here.